feature-libs/my-account/organization/src/components/user-group/form/user-group-form.module.ts

```diff
@@ -17,12 +21,11 @@ import { FormErrorsModule } from '../../../../shared/components/form/form-errors
     UrlModule,
     I18nModule,
     ReactiveFormsModule,
-    DatePickerModule,
     FormErrorsModule,
   ],
   declarations: [UserGroupFormComponent],
   exports: [UserGroupFormComponent],
-  providers: [OrgUnitService],
+  providers: [CurrencyService, OrgUnitService],
```

Comment:
Do we really need to provide `CurrencyService` here?

Reply:
no